pkg/reconciler/taskrun/resources/image_exporter_test.go

```diff
@@ -167,15 +163,15 @@ func TestAddOutputImageDigestExporter(t *testing.T) {
 	}} {
 		t.Run(c.desc, func(t *testing.T) {
 			names.TestingSeed()
-			gr := func(n string) (*v1alpha1.PipelineResource, error) {
-				return &v1alpha1.PipelineResource{
+			gr := func(n string) (*resourcev1alpha1.PipelineResource, error) {
+				return &resourcev1alpha1.PipelineResource{
 					ObjectMeta: metav1.ObjectMeta{
 						Name:      "source-image-1",
 						Namespace: "marshmallow",
 					},
-					Spec: v1alpha1.PipelineResourceSpec{
+					Spec: resourcev1alpha1.PipelineResourceSpec{
 						Type: "image",
-						Params: []v1alpha1.ResourceParam{{
+						Params: []v1beta1.ResourceParam{{
```

Comment:
Should this be `Params: []resourcev1alpha1.ResourceParam` instead of  `Params: []v1beta1.ResourceParam`? 🤔